securedrop/sdconfig.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+import config as _config
+
+MYPY = False
+if MYPY:
+    from typing import List  # noqa: F401
+
+
+class SDConfig(object):
+    def __init__(self):
+        # type: () -> None
+        self.ADJECTIVES = _config.ADJECTIVES
+        self.DATABASE_ENGINE = _config.DATABASE_ENGINE
+        self.DEFAULT_LOCALE = _config.DEFAULT_LOCALE
```

Comment:
`DEFAULT_LOCALE` may not be defined in instances installed prior to 0.4.3

Reply:
Yes, this is taken care of in the current update.